jina/executors/evaluators/craft/nlp/__init__.py

```diff
@@ -0,0 +1,21 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+from typing import Any
+from .. import BaseEvaluator
+
+
+class BaseNlpCraftingEvaluator(BaseEvaluator):
```

Comment:
what's the difference between `BaseCraftingEvaluator ` and `BaseNlpCraftingEvaluator `?